src/scss/attendee/effects/error.scss

```diff
@@ -0,0 +1,4 @@
+@mixin input-error {
```

Comment:
This doesn't look like a file for these values. There is a `theme.scss` for the colors that should allow for the switch dark/light mode (and keep all color values in the same place) which other files in `src/scss/attendee/effects` use.

Reply:
imported values from `theme` [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)